### DISCUSSION TITLE
How can I hide this error?

### DISCUSSION THREAD

**Discussion by ifconfig2333** [pyinstaller/pyinstaller#6743] [Help]
I don't want this program to pop up this window

How to hide him?

![图片](https://user-images.githubusercontent.com/43915014/163020041-a29fb9c2-1e7d-48b5-8ed7-7fac0df68bac.png)

I don't want him to pop up this window

 hide . help

**Comment by rokm** [comment]
Implement proper exception handling in your program.

**Comment by momolly1024** [comment]
I have the same question too.
Is there any way to hide the error when run exe failed ?

**Reply by rokm** [reply]
Catch and handle your exceptions like you should, and the unhandled exception dialog won't be shown.
